requirements/udata.in

```diff
@@ -1 +1 @@
-udata@git+https://github.com/opendatateam/udata.git
+https://21428-19141754-gh.circle-artifacts.com/0/dist/udata-3.1.1.dev21428-py2.py3-none-any.whl
```

Comment:
next time we could try to point to the git branch, just for fun

Reply:
(I actually tried pinning a commit, with success)